backend/tests/endpoints/v1/test_jwt_auth.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+from faker import Faker
+from fastapi import status
+from httpx import AsyncClient
+
+from src.models import User
+from src.utils import url_for
+from tests.factories.schemas import UserRegisterSchemaFactory
+
+fake = Faker()
+
+
+@pytest.mark.asyncio
+async def test_register(http_client: AsyncClient):
```

Comment:
Instead of marking each test with @pytest.mark.asyncio you can provide default pytest configuration in the pyproject.toml file to be in async mode by default